server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -1902,13 +1902,10 @@ public void removeCustomOfferingDetails(long vmId) {
     }
 
     @Override
-    public HashMap<String, VolumeStatsEntry> getVolumeStatistics(long clusterId, String poolUuid, StoragePoolType poolType, List<String> volumeLocator, int timeout) {
+    public HashMap<String, VolumeStatsEntry> getVolumeStatistics(long clusterId, String poolUuid, StoragePoolType poolType, List<String> volumeLocator) {
         List<HostVO> neighbors = _resourceMgr.listHostsInClusterByStatus(clusterId, Status.Up);
         for (HostVO neighbor : neighbors) {
             GetVolumeStatsCommand cmd = new GetVolumeStatsCommand(poolType, poolUuid, volumeLocator);
-            if (timeout > 0) {
```

Comment:
Well looks like it indeed is used but you removed it @GabrielBrascher ?

Reply:
@rhtyd the timeout is used to set the `GetVolumeStatsCommand` `wait` variable. However, I was not able to find any piece of code using the `wait` value, neither on `LibvirtGetVolumeStatsCommandWrapper` and `CitrixGetVolumeStatsCommandWrapper`.